main.py

```diff
@@ -2,6 +2,43 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    create_genres(("Western", "Action", "Dramma"))
+    create_actors([("George", "Klooney"), ("Kianu", "Reaves"),
```

Comment:
Not a mistake but I guess here you could put this list in variable and put each tuple on new line for better readability